.github/workflows/ci.yml

```diff
@@ -59,7 +59,8 @@ jobs:
           mysql -h localhost -P 3306 --protocol=tcp -u root --password=password < scripts/inserts.sql
       - name: Test with pytest
         run: |
-          pytest tests --verbose --failed-first
+          pip3 install -r requirements.txt
```

Comment:
Does test_routes.py only need the dependencies related to testing?
If that's the case, add the only to requirements-dev.txt.

requirements.txt is for dependencies that require for the asset application to run. No need to add testing related dependencies to a prod, docker container.

Reply:
These tests are not running against the docker instance. It runs the app separately and tests the endpoints. So the tests fail if the dependencies are missing.